src/virtual-source-buffer.js

```diff
@@ -437,10 +437,6 @@ export default class VirtualSourceBuffer extends videojs.EventTarget {
     }
 
     if (this.appendInitSegment_) {
```

Comment:
Maybe we can handle it in a different PR, but we should probably change this to appendAudioInitSegment_ now.